tests/testsuite/read_manifest.rs

```diff
@@ -274,3 +274,502 @@ fn workspace_package() {
+        "links": null,
+        "metabuild": null,
+        "metadata": null,
+        "name": "crate2",
+        "publish": null,
+        "readme": false,
+        "repository": null,
+        "resolver": null,
+        "rust-version": null,
+        "version": "0.1.0",
+        "workspace": null
+      },
+      "patch": null,
+      "profile": null,
+      "project": null,
+      "replace": null,
+      "target": null,
+      "test": [],
+      "workspace": null
+    },
+    "reason": "manifest"
+  },
+  {
+    "manifest": {
+      "badges": null,
+      "bench": [],
+      "bin": [
+        {
+          "bench": null,
+          "crate-type": null,
+          "crate_type": null,
+          "doc": null,
+          "doc-scrape-examples": null,
+          "doctest": null,
+          "edition": null,
+          "filename": null,
+          "harness": null,
+          "name": "read-manifest-test",
+          "path": "src/main.rs",
+          "proc-macro": null,
+          "proc_macro": null,
+          "required-features": null,
+          "test": null
+        }
+      ],
+      "build-dependencies": null,
+      "build_dependencies": null,
+      "cargo-features": null,
+      "dependencies": null,
+      "dev-dependencies": null,
+      "dev_dependencies": null,
+      "example": [],
+      "features": null,
+      "lib": null,
+      "lints": null,
+      "package": {
+        "authors": [],
+        "autobenches": false,
+        "autobins": false,
+        "autoexamples": false,
+        "autolib": false,
+        "autotests": false,
+        "build": false,
+        "categories": null,
+        "default-run": null,
+        "default-target": null,
+        "description": null,
+        "documentation": null,
+        "edition": "2024",
+        "exclude": null,
+        "forced-target": null,
+        "homepage": null,
+        "im-a-teapot": null,
+        "include": null,
+        "keywords": null,
+        "license": null,
+        "license-file": null,
+        "links": null,
+        "metabuild": null,
+        "metadata": null,
+        "name": "read-manifest-test",
+        "publish": null,
+        "readme": false,
+        "repository": null,
+        "resolver": null,
+        "rust-version": null,
+        "version": "0.1.0",
+        "workspace": null
+      },
+      "patch": null,
+      "profile": null,
+      "project": null,
+      "replace": null,
+      "target": null,
+      "test": [],
+      "workspace": {
+        "default-members": null,
+        "dependencies": null,
+        "exclude": null,
+        "lints": null,
+        "members": [
+          "crate1",
+          "crate2"
+        ],
+        "metadata": null,
+        "package": null,
+        "resolver": "3"
+      }
+    },
+    "reason": "manifest"
+  }
+]
+"#]]
+            .is_json()
+            .against_jsonlines(),
+        )
+        .with_status(0)
+        .run();
+}
+
+#[cargo_test]
+fn read_manifests_in_workspace_2() {
```

Comment:
Let's not just do a `2` in a test name.  What is it you are trying to cover in this test that isn't in the previous test?